Dockerfile

```diff
@@ -294,6 +295,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         jq \
         less \
         libgomp1 \
+        libopenblas-base \
```

Comment:
> Is cvxopt also going to need SuiteSparse at runtime? (since that's also necessary at compile time)

Good point. `import cvxopt` works now, but that exact command might not require SuiteSparse as it does with LAPACK. I'll look for a command that depends on SuiteSparse being present and test with that.

> Separately, can we include these libraries only for arm64 and not for amd64 where they're unneeded/unused?

👍

Reply:
> Is cvxopt also going to need SuiteSparse at runtime? (since that's also necessary at compile time)

I'm not sure how to test that CVXOPT works with SuiteSparse or not, but I added c4650eb8542ae7fa394bad529bb243592c02badd to address this.

> Separately, can we include these libraries only for arm64 and not for amd64 where they're unneeded/unused?

Done in fa48e73b761dbc7c55206d24d5e4f0436a77100d.